src/containers/Home/Home.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Divider,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Box
+} from '@material-ui/core';
+import { NoteAdd as NoteAddIcon } from '@material-ui/icons';
+import { API } from 'aws-amplify';
+
+import { useHomeStyle } from './Home.style';
+
+export default function Home(props) {
+  const classes = useHomeStyle();
+  const [notes, setNotes] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    async function onLoad() {
```

Comment:
why do you need to create new `onLoad()` every time?

Reply:
Hey @xuatz

Sorry, I didn't change the logic of the application boilerplate currently. So there are questionable logic in the current state of the app. They will be moved out after I integrated `redux/redux-saga`